SmartDeviceLink/SDLAudioStreamManager.m

```diff
@@ -38,7 +39,7 @@ - (instancetype)initWithManager:(id<SDLStreamingAudioManagerType>)streamManager
     if (!self) { return nil; }
 
     _mutableQueue = [NSMutableArray array];
-    _audioQueue = dispatch_queue_create("com.sdl.audiomanager.transcode", DISPATCH_QUEUE_SERIAL);
+    _audioQueue = dispatch_queue_create_with_target("com.sdl.audiomanager.transcode", DISPATCH_QUEUE_SERIAL, [SDLGlobals sharedGlobals].sdlProcessingQueue);
```

Comment:
When I install the library in a project I get this warning:
> 'dispatch_queue_create_with_target' is only available on iOS 10.0 or newer

When I click on the warning for more detail I get:

> 'dispatch_queue_create_with_target' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 8.0.0

Reply:
It should be fixed now